packages/stryker/test/unit/reporters/ci/TravisProviderSpec.ts

```diff
@@ -0,0 +1,70 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import * as environmentVariables from '../../../../src/utils/objectUtils';
+
+import TravisProvider from '../../../../src/reporters/ci/TravisProvider';
+
+describe('Travis Provider', () => {
```

Comment:
This is OK for now, but we might want to consider moving this to a more DRY approach. Defining the specs in a function and calling that function from both the `TravisProviderSpec` and the `CircleProviderSpec`.